README.md

```diff
@@ -17,3 +17,14 @@ This is the current architecture of our program. Each block represents a collect
 # Helpful Commands
 `go run .`   
 `go build .`
+
+# GitHub API Notes
+endpoints : /license, /responsiveness, /contributors
+need to add /correctness, etc.
+
+using localhost:3000
+
+e.g. to add headers to get terminal responses
+curl -H "InputURL: https://github.com/qiangxue/go-rest-api" -H "GitHubToken: <YOURGPAT>" localhost:3000/api/license
```

Comment:
If I’m reading this right, it looks like you built a web server to handle requests from our CLI to get metric data. Is there a reason you didn’t just call your functions directly from the CLI? In my experience that would be much faster and require less resources.

This seems like a design change that we should talk about as a group.

Reply:
So what we can do is make a wrapper for curl commands that'll let us call it from the CLI instead